### DISCUSSION TITLE
How do I set up APKs? if at all?

### DISCUSSION THREAD

**Discussion by tomashaha** [dnschneid/crouton#4596] [Q&A]
Hey, so I just got an old fashioned chromebook, only wants to update to version 76 because I'm a broke teenager. I was wondering if its possible, with the help of crouton, to sideload apks, without use of the appstore. When I try to look it up all it shows me is just laughing in my face about the new chromebooks. 

Sooooo, I'm trying to run citra, and a few management apps that only work on my phone. Is it possible to load those apks onto my laptop, and boot em up using crouton.

redundant I know

**Comment by DennisLfromGA** [comment]
@tomashaha,

*If* your 'old fashioned' Chromebook supports the Google Play store just get into Developer mode, download an apk file, right-click on it and select 'Package installer'.

If it does not support Google Play store then you may be able to install '[brunch](https://github.com/sebanc/brunch#hardware-support-and-added-features)' on it but that depends on your hardware.

*Hope this helps,*
-**DennisLfromGA**

**Comment by ghost** [comment]
If by what you mean "old fashioned" is "my Chromebook doesn't support Android apps" then there are a couple of things you can try:

 - Anbox
 - https://forum.xda-developers.com/t/chromebook-acer-c720p-getting-google-play-store-with-chrome-os-65-recovery-from-usb.3805304/
 - https://archon-runtime.github.io/
 - There was an app on the Chrome Web Store similar to ARChon, although I couldn't find it. 

Personally I haven't had amazing experiences with any of these.
